docs/testing.md

```diff
@@ -92,5 +86,5 @@ gcloud builds submit --tag \
 
 ## Terraform Testing
```

Comment:
remove: there is no terraform testing presently

Reply:
Removed.